numpy/_core/src/multiarray/getset.c

```diff
@@ -116,6 +116,10 @@ array_strides_get(PyArrayObject *self, void *NPY_UNUSED(ignored))
 static int
 array_strides_set(PyArrayObject *self, PyObject *obj, void *NPY_UNUSED(ignored))
 {
+    if( DEPRECATE("Setting the strides on a NumPy array has been deprecated in NumPy 2.3.") < 0 ) {
```

Comment:
It may be nice to point out the work-arounds, but considering that stride setting is a bit niche maybe we can also just do that in the release note.

Something like: See `np.lib.stride_tricks.strided_window_view` and `np.lib.stride_tricks.as_strided`.

Reply:
And here is a long line, which need space fixes also.

Reply:
If you are at it, adding a `/* Deprecated NumPy 2.3, 2025-05-11 */` above it would also be nice.